web_accounts_list.json

```diff
@@ -4591,6 +4591,61 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "Tunisie-Foot",
+           "check_uri" : "https://www.tunisie-foot.com/forum/wcf/user/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "profile",
+           "account_missing_string" : "Page Non Trouvée - Tunisie-Foot",
+           "account_missing_code" : "404",
+           "known_accounts" : ["7701-manager", "28487-tunisia-gang-city"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "ICQ Chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "person",
+           "account_missing_string" : "Oops! We ran into some problems",
+           "account_missing_code" : "404",
+           "known_accounts" : ["brookenora.54", "bigdaddy.77"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "HeyLink.me",
+           "check_uri" : "https://heylink.me/{account}",

```

Comment:
Here too, you need to add a trailing `/` to the end of the `check_uri` as the browser/checker gets a 301 and redirected.